src/backend/bsd.js

```diff
@@ -17,6 +18,19 @@ export default class BSD {
     this.apiSecret = secret;
   }
 
+  noFailApiRequest(method, ...args) {
+    try {
+      BSD[method](...args);
+    } catch (e) {
+      GCAudit.create({
+        class: 'BSDClient',
+        method: method,
+        params: String(args),
```

Comment:
We may want to automatically retry these.  Is String good enough for serialization?